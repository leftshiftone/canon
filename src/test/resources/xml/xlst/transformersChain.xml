<markup>
    <container>
        <text id="123">Basierend auf Ihren Angaben k√∂nnen wir Ihnen folgende ${result.size()} Resultate vorschlagen:</text>
        <carousel>
            <block foreach="$lawyer in $result">
                <headline>{{$lawyer.name}}</headline>
                <text>${lawyer.address}</text>
                <textInput name="textInput" placeholder="type here..." regex="" value="foo" required="true" class="text"/>
                <textA>A</textA>
                <textB>B</textB>
                <textC>C</textC>
            </block>
        </carousel>
    </container>
</markup>