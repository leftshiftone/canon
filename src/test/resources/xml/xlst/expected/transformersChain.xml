<markup>
    <container>
        <text id="123">Basierend auf Ihren Angaben k√∂nnen wir Ihnen folgende ${result.size()} Resultate vorschlagen:</text>
        <carousel>
            <block foreach="$lawyer in $result">
                <headline>{{$lawyer.name}}</headline>
                <label>${lawyer.address}</label>
                <text name="textInput" placeholder="type here..." regex="" value="foo" required="true" class="text"/>
                <labelA>A</labelA>
                <labelB>B</labelB>
                <labelC>C</labelC>
            </block>
        </carousel>
    </container>
</markup>